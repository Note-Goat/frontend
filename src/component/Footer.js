import {Link} from "react-router-dom";

export default function Footer() {
  return (
    <div className="footer-container">
      <div className="footer-controls">
        <Link to="/privacy">Privacy</Link>
        <Link to="/contact">Contact</Link>
      </div>
    </div>
  );
}
