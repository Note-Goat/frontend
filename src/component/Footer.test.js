import renderer from 'react-test-renderer';
import Footer from "./Footer";
import {BrowserRouter} from "react-router-dom";

test('Footer component snapshot', () => {
  const tree = renderer
    .create(<BrowserRouter><Footer /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
