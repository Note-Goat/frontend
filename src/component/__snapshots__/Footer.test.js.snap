// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component snapshot 1`] = `
<div
  className="footer-container"
>
  <div
    className="footer-controls"
  >
    <a
      href="/privacy"
      onClick={[Function]}
    >
      Privacy
    </a>
    <a
      href="/contact"
      onClick={[Function]}
    >
      Contact
    </a>
  </div>
</div>
`;
